### ISSUE TITLE
Support of other types of devices, custom icon for states other than normal?

### ISSUE THREAD

**Issue by gazedash** [verypossible/rn-custom-airplay#1]
I gotta say a really nice job especially a library in Swift, but what about the other types of devices?
E.g. bluetooth, headphones etc

What about other button states?

I implemented this in my library https://github.com/gazedash/react-native-airplay-ios

We can cooperate and maybe make AirPlay support on React Native platform better, what do you think?

**Comment by Yunnathai** [comment]
thank your work. what about android like DLNA？

**Comment by gazedash** [comment]
@jamninetyfive I believe this library has nothing to do with DLNA and it’s out of scope.

**Comment by Yunnathai** [comment]
> @jamninetyfive I believe this library has nothing to do with DLNA and it’s out of scope.

Thank you for your explanation.
 i want know this library can do such a thing?
use react-native-video play video then throwing screen to TV

**Comment by gazedash** [comment]
@jamninetyfive No, it doesn't support DLNA. It is specifically made for AirPlay which you can use to stream specifically to Apple TV only.

**Comment by Yunnathai** [comment]
> @jamninetyfive No, it doesn't support DLNA. It is specifically made for AirPlay which you can use to stream specifically to Apple TV only.

OK thx

**Comment by Yunnathai** [comment]
> @jamninetyfive No, it doesn't support DLNA. It is specifically made for AirPlay which you can use to stream specifically to Apple TV only.

I need a library when I use react-native-video to play videos.
Supporting AirPlay Connection to TV via DLNA or AirPlay. Not Apple TV.maybe XIAOMI TV LECOO TV 
What should I do? Can you help me?

**Comment by Mrjaco12** [comment]
@gazedash thanks for the link. We don't currently have any type of roadmap for this package as it meets our current needs but I think we'd definitely be open to adding some of the things you mentioned.
